src/generators/javascript/JavaScriptGenerator.ts

```diff
@@ -27,17 +27,17 @@ export class JavaScriptGenerator extends AbstractGenerator<JavaScriptOptions> {
     super('JavaScript', JavaScriptGenerator.defaultOptions, options);
   }
 
-  async render(model: CommonModel, inputModel: CommonInputModel): Promise<string> {
+  render(model: CommonModel, inputModel: CommonInputModel): Promise<string> {
```

Comment:
Shouldnt be here `: string` instead of `: Promise<string>`?

Reply:
I see that this is repeated in many more places.

Reply:
It is because we wanted to allow the async `presets` that we used `Promise` throughout 🙂